scripts/convert.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+
+import csv
+import json
+import os
+
+
+def csv_to_json(dir_path, filename, output_dir_path=None, labels=[]):
+    """Converts CSV data file to JSON data file
+
+    dir_path -- path to directory containing CSV file
+    filename -- CSV file name
+    output_dir_path -- output directory where JSON file will be placed
+    """
+    if output_dir_path is None:
+        return
```

Comment:
Why are you creating a default argument, when obviously it should be obligatory?

Also for future, when checking arguments it would be better to fail with an exception rather than do it silently.